### ISSUE TITLE
Use a title attribute to help explain lightning bolt in UI

### ISSUE THREAD

**Issue by cfjedimaster** [strongloop/strong-arc#1719]
I've got something in Composer I've never noticed before:

![image](https://cloud.githubusercontent.com/assets/393660/13053341/9aaf89d4-d3c9-11e5-88c5-4740cd66a7cc.png)

See the lightning bolt? I assume it implies something about this model, but the image doesn't make use of an ALT or TITLE attribute to provide context about what warning (I assume) this is representing. (And by inspecting, I see it is a CSS class adding it, not a raw <img> tag, but still, it should explain itself. :)

**Comment by widmur** [comment]
Agreed. I'm experiencing the same issue and have no idea what it means. Any developments?
